backend/curation/api/curation-api.yml

```diff
@@ -14,6 +14,45 @@ info:
     ## Data Model
     <!-- Data model diagram created via Lucid: https://lucid.app/lucidchart/8d085832-34f4-4413-9c1d-44e77d06e326/edit?invitationId=inv_ed354532-7256-4e0a-9ad0-b622905e5198&page=Vfzkzdv~I1fL# -->
     <img src="/static/discover-api-data-model.png" width="50%"/>
+
+    ## Working with versions
+
+    #### Access to prior published version history
+    The CZ CELLxGENE Discover API employs the concept of *versions* for our two core entities, *Collections* and
+    *Datasets*. This is to permit access to prior published instances (versions) of these entities even as they are
+    changed over time via revisions (at present the Data Portal web application UI offers access to only the latest
```

Comment:
I would remove this reference to how Data Portal UI handles revisions--it's not related to curation API (and its also not true yet)

Reply:
Hold on. It is true. The Data Portal UI does not permit access to past versions.

Reply:
Do you mean you want me to remove the parenthetical note? Or the entire sentence?

Reply:
Just remove the parenthetical note.

Also, you can put a collection version ID in the URL of the Data Portal UI and it'll render that prior published version. Albeit there is a ticket to redirect, its not implemented yet

Ex. 
https://api.cellxgene.dev.single-cell.czi.technology/curation/v1/collections/5e512a4a-4f42-4c1a-8220-cfb9df2e7421/versions

https://cellxgene.dev.single-cell.czi.technology/collections/5e512a4a-4f42-4c1a-8220-cfb9df2e7421

https://cellxgene.dev.single-cell.czi.technology/collections/eacc1698-85f7-4583-bc12-1668af7613f3

Reply:
Interesting... ok

Got it, we'll skip this.